liqd_product/templates/header.html

```diff
@@ -10,21 +10,26 @@
         </div>
         <nav class="main-header__change-me">
             {% if PARTNER %}
-                <a href="{{ PARTNER.get_absolute_url }}" class="main-header__logo-box" rel="home">
+                <a href="{% url 'partner' partner_slug=PARTNER.slug %}" class="main-header__logo-box" rel="home">
                     <img src="{% thumbnail PARTNER.logo '0x160' %}" height="80" alt="{{ PARTNER.name }}" class="main-header__logo-img" />
                 </a>
             {% endif %}
             <div class="main-header__mobile-menu collapse" id="main-header-collapse">
                 {% if PARTNER %}
                     <ul class="main-header__partner-nav">
                         <li class="main-header__partner-nav__item">
-                            <a href="#" class="main-header__partner-nav__link">{% trans 'Beteiligen' %}</a>
+                            {% url 'partner' partner_slug=PARTNER.slug as partner_url %}
+                            <a href="{{ partner_url }}" class="main-header__partner-nav__link"><span class="{% if request.path == partner_url %}is-active{% endif %}">{% trans 'Beteiligen' %}</span></a>
                         </li>
+                        {% if PARTNER.information %}
                         <li class="main-header__partner-nav__item">
-                            <a href="{% url 'partner-information' partner_slug=PARTNER.slug %}" class="main-header__partner-nav__link">{% trans 'FAQ' %}</a>
+                            {% url 'partner-information' partner_slug=PARTNER.slug as partner_information_url %}
+                            <a href="{{ partner_information_url }}" class="main-header__partner-nav__link"><span class="{% if request.path == partner_information_url %}is-active{% endif %}">{% trans 'FAQ' %}</span></a>
                         </li>
+                        {% endif %}
                         <li class="main-header__partner-nav__item">
```

Comment:
Here is an {% if partner.imprint %} missing, or not?

Reply:
the imprint should not be optional. only the information